### ISSUE TITLE
What does the various  column items in det.txt file represent?

### ISSUE THREAD

**Issue by Akola-Mbey-Denis** [phil-bergmann/tracking_wo_bnw#160]
Hello,
I am trying to understand the code of  your tracktor.  I  would like to know how the det file is generated and possibly  if the image name could be generated from it? Looking forward to hear from you

**Comment by Akola-Mbey-Denis** [comment]
Hello all,can you generate gt.txt from det data?

**Comment by timmeinhardt** [comment]
No, both the `det.txt` and `gt.txt` are provided by the MOT17 dataset.

**Comment by Akola-Mbey-Denis** [comment]
Ok,thats,please how do you generate annotations for the MOT17 dataset. Specifically,the test dataset?

**Comment by timmeinhardt** [comment]
You do not generate annotations. These are provided by the dataset. However, only for the training sequences. If yoy want to evaluate test set results you need to create an account on the MOTChallenge webpage and submit predicted results to the evaluation server.

**Comment by Akola-Mbey-Denis** [comment]
Ok,thanks,you mean if I download the data from MOT Challenge website both the train and test data have their annotation.json file embedded in them?

**Comment by timmeinhardt** [comment]
No, you will never get the test set annotations. These are not public. You need to submit your predicted tracking results and then get an email with your results from the evaluation server.

**Comment by Akola-Mbey-Denis** [comment]
Alright thanks

**Comment by Akola-Mbey-Denis** [comment]
Please can I get your email ?

**Comment by timmeinhardt** [comment]
What do you mean? My email address is shown on my profile page.
